editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -56,8 +60,7 @@ function Component(props: Props) {
 
   // PlanningConstraints must come after at least a FindProperty in the graph
   const showGraphError = !longitude || !latitude;
-  if (showGraphError)
-    throw new GraphError("mapInputFieldMustFollowFindProperty");
+  if (showGraphError) throw new GraphError("nodeMustFollowFindProperty");
```

Comment:
This is a small unrelated bug fix - this was incorrectly displaying the Map & Label "schema" graph error, when it should be a more generic sequence error